news/4.37/examples/terminal/.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>test.terminal</name>
```

Comment:
This is a strange name for a project.  I expect the oomph setup will import this and we probably won’t want that.